src/Debug.h

```diff
@@ -96,6 +97,8 @@ class Debug
     /// logs output buffer created in Start() and closes debugging context
     static void Finish();
 
+    /// Configures the active debugging context to write syslog ALERT
```

Comment:
s/Configures/configures/ so that you do not have to put a period at the end and for consistency sake.